.gitignore

```diff
@@ -14,16 +14,11 @@ venv*
 *.pyc

```

Comment:
The changes in `.gitignore` remove unnecessary whitespace and some entries. Ensure that removing these entries (like `~` and `core/build/lib/*`) won't affect other developers or builds.